core/src/main/java/tc/oc/pgm/spawner/SpawnerModule.java

```diff
@@ -83,10 +87,55 @@ public SpawnerModule parse(MapFactory factory, Logger logger, Document doc)
             XMLUtils.getChildren(
                 element, "item")) { // TODO Add more types of spawnables once entity parser is built
```

Comment:
```suggestion
            XMLUtils.getChildren(element, "item")) {
```
This todo becomes outdated i guess :)